src/modules/hotkeys/assets/commands/mission.ts

```diff
@@ -85,4 +89,98 @@ export default <Scope<Empty, ['sorted'], [], true>>{
             }
         },
     },
+    alliance: <
+        Scope<
+            {
+                responseInputbox: HTMLInputElement | null;
+                responseCheckbox: HTMLInputElement | null;
+            },
+            [],
+            ['focus', 'toggle']
+        >
+    >{
+        validatorFunction() {
+            this.responseInputbox = document.querySelector<HTMLInputElement>(
+                '#mission_reply_content'
+            );
+            this.responseCheckbox = document.querySelector<HTMLInputElement>(
+                '#mission_reply_alliance_chat'
+            );
+            return !!this.responseInputbox && !!this.responseCheckbox;
+        },
+        toggle() {
+            if (this.responseCheckbox)
+                this.responseCheckbox.checked = !this.responseCheckbox.checked;
+        },
+        focus() {
+            this.responseInputbox?.focus();
+        },
+    },
+    arr: <
+        Scope<{ arrList: HTMLUListElement | null }, [], ['next', 'previous']>
+    >{
+        validatorFunction() {
+            this.arrList =
+                document.querySelector<HTMLUListElement>('#aao-tabs');
+            return !!this.arrList;
+        },
+        next() {
+            const current = this.arrList?.querySelector('.active');
+            //Check whether this is the last element
+            if (current?.nextElementSibling != null) {
+                const next = current?.nextElementSibling?.firstElementChild;
+                (next as HTMLElement).click();
+            }
+        },
+        previous() {
+            const current = this.arrList?.querySelector('.active');
+            //Check whether this is the first element
+            if (current?.previousElementSibling != null) {
+                const previous =
+                    current?.previousElementSibling?.firstElementChild;
+                (previous as HTMLElement).click();
+            }
+        },
+    },
+    vehicleList: <
+        Scope<
+            {
+                vehicleList: HTMLUListElement | null;
+                loadMissingBtn: HTMLAnchorElement | null;
+            },
+            [],
+            ['next', 'previous', 'loadMissing']
+        >
+    >{
+        validatorFunction() {
+            this.vehicleList =
+                document.querySelector<HTMLUListElement>('#tabs');
+            this.loadMissingBtn = document.querySelector<HTMLAnchorElement>(
+                '.missing_vehicles_load'
+            );
+            return !!this.vehicleList && !!this.loadMissingBtn;
```

Comment:
This will return `false` when the loadMissingBtn is not rendered due to the player not having > 250 vehicles in Range. You may simulate that by setting your vehicle range to a low value such as 1km.
=> vehicleList will not work if loadMissingBtn does not exist.
Consider removing the check from validator function and instead checking for existance in the methods that use loadMissingBtn

Reply:
Fixed in [d7a8d53](https://github.com/LSS-Manager/LSSM-V.4/pull/1717/commits/d7a8d53e6c12024ab560fb108f86293bf001a332)